libs/list-types/common/src/sjp/json-parser.ts

```diff
@@ -0,0 +1,275 @@
+import type { SjpCasePress, SjpCasePublic } from "./sjp-service.js";
+
+export interface SjpJson {
```

Comment:
This feels wrong to sit in common due to the SJP specific nature of the functions. I think this should sit in the list specific folders instead